server_test.go

```diff
@@ -1036,6 +1036,11 @@ func NewMessagingClient() *MessagingClient {
 	return c
 }
 
+// Opened returns true if the messaging client has been opened
+func (c *MessagingClient) Opened() bool {
```

Comment:
Since this doesn't do much, it seems like if a client exists at all, it's ready?